addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/handler/LoxoneMiniserverHandler.java

```diff
@@ -331,24 +333,53 @@ public void onServerGoesOffline(LxOfflineReason reason, String details) {
                 updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "Unknown reason");
                 break;
         }
-
     }
 
     @Override
     public void dispose() {
         logger.debug("Disposing of server");
-        if (server != null) {
-            server.stop();
+        LxServer srv = server;
```

Comment:
Why are you copying the class variable into a local one before checking for `null` and stopping the server?